### PR TITLE
docs: ✏️ Add backtick around <T> and export missing v2 types

### PR DESCRIPTION
**Pull request by prashantasdeveloper**
### Description

Adds `\` for proper rendering of <T> in docs + exports missing v2 types needed in documentation

### Breaking Changes

NA

### JIRA Link

NA

### Checklist

- [ ] Updated the Readme.md (if required) ?

### CODE REVIEW

File: src/types/utils/index.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ export type UnionOfProcedureFuncs<Args, ReturnValue, Storage> = Args extends unk
   : never;
 
 /**
- * Less strict version of Parameters<T>
+ * Less strict version of Parameters<T\>
```
Comment (polymath-eric) [inline src/types/utils/index.ts:27]:
```suggestion
 * Less strict version of `Parameters<T>`
```

I thought we wanted to use `

Comment (prashantasdeveloper) [reply src/types/utils/index.ts:27]: Ohh when I checked and generated docs, it altogether skipped <T> and only had `Less strict version of Parameters` so I thought all we wanted was to display it correctly

**Review by polymath-eric — APPROVED** [review]

**Pull request merged**